src/Symfony/Component/Security/Core/Tests/Authorization/Voter/AbstractVoterTest.php

```diff
@@ -59,86 +76,55 @@ public function testVoteUsingDeprecatedIsGranted($expectedVote, $object, $attrib
      */
     public function testNoOverriddenMethodsThrowsException()
     {
-        $voter = new DeprecatedVoterNothingImplementedFixture();
-        $voter->vote($this->token, new ObjectFixture(), array('foo'));
-    }
+        $voter = new AbstractVoterTest_NothingImplementedVoter();
 
-    public function getData()
-    {
-        return array(
-            array(AbstractVoter::ACCESS_ABSTAIN, null, array(), 'ACCESS_ABSTAIN for null objects'),
-            array(AbstractVoter::ACCESS_ABSTAIN, new UnsupportedObjectFixture(), array(), 'ACCESS_ABSTAIN for objects with unsupported class'),
-            array(AbstractVoter::ACCESS_ABSTAIN, new ObjectFixture(), array(), 'ACCESS_ABSTAIN for no attributes'),
-            array(AbstractVoter::ACCESS_ABSTAIN, new ObjectFixture(), array('foobar'), 'ACCESS_ABSTAIN for unsupported attributes'),
-            array(AbstractVoter::ACCESS_GRANTED, new ObjectFixture(), array('foo'), 'ACCESS_GRANTED if attribute grants access'),
-            array(AbstractVoter::ACCESS_GRANTED, new ObjectFixture(), array('bar', 'foo'), 'ACCESS_GRANTED if *at least one* attribute grants access'),
-            array(AbstractVoter::ACCESS_GRANTED, new ObjectFixture(), array('foobar', 'foo'), 'ACCESS_GRANTED if *at least one* attribute grants access'),
-            array(AbstractVoter::ACCESS_DENIED, new ObjectFixture(), array('bar', 'baz'), 'ACCESS_DENIED for if no attribute grants access'),
-        );
+        $voter->vote($this->token, $this->object, array('EDIT'));
     }
 }
 
-class VoterFixture extends AbstractVoter
+class AbstractVoterTest_Voter extends AbstractVoter
```

Comment:
wouldn't it be possible to keep the existing name for the class, to make merging between 2.7 and 2.8 easier for the next 3 years ?